### ISSUE TITLE
Issue with onChange callback

### ISSUE THREAD

**Issue by dashersch** [michelson/dante2#207]
I want to  catch any edits in Dante2 and write them to console as described [here](https://michelson.github.io/dante2/#onchange-callback) 

This is not working. I am not getting any output to the console.

My set up is:
I used `npm install --save Dante2@next` to install and my JS source looks like

```javascript
import Layout from '../components/MyLayout';
import Dante from "Dante2"

export default function Editor() {
    return (
      <div>
        <Layout>
          
        <Dante onChange={editor => { console.log('editor content: ', editor.emitSerializedOutput()) }}/>

        </Layout>
      </div>
    );
  }
```

**Comment by mbm-michal** [comment]
@dashersch I have:
```js
<Dante
	data_storage={{
	   save_handler: this.onSaveHandler,
	}}
/>
```
 and then I use redux to save to state 
```js
onSaveHandler = (editorContext, content) => {
		const { id } = this.props;
		this.props.updateSectionContent({
			[id]: draftIncludeMarkdown(content),
		});
	};
```
hope that helps!

**Comment by dashersch** [comment]
Thanks @michelson  I will try out your suggestion.

In the meantime, I saw in the logs that I was missing prism. Running `npm install --save prismjs@1.x` installed prism and now the method described in your documentation for onChange works as well

**Comment by michelson** [comment]
Good, also you can implement the data storage `save_handler`

data_storage={
{
url: "/",
save_handler: this.saveHandler
}
}


note that the `url: /` is needed

Atte.
Miguel Michelson Martinez
www.vadb.org <http://vadb.org>

**Comment by chfritz** [comment]
I had the issue (the `onChange` handler wasn't firing). I was able to get it to work by *not* following the install instructions but instead running:
```
npm install --save Dante2
```
without the `@next`. This installed version 0.5.0-rc9 instead of rc4. After that, as @dashersch already reported, I had to install prismjs as well (`npm install --save prismjs`).

Now the onChange handler is firing happily.
